packages/dd-trace/src/appsec/sdk/noop.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+class AppsecSdk {
+  trackUserLoginSuccessEvent (user, metadata) { }
+
+  trackUserLoginFailureEvent (userId, exists, metadata) { }
+
+  trackCustomEvent (eventName, metadata) { }
+}
+module.exports = AppsecSdk
```

Comment:
- Change name of class to add the Noop name, useful for debugging
- No need for the arguments I think, especially since we'll forget to update them later
- whitespace changes
```suggestion
class NoopAppsecSdk {
  trackUserLoginSuccessEvent () {}

  trackUserLoginFailureEvent () {}

  trackCustomEvent () {}
}

module.exports = NoopAppsecSdk
```